### PR TITLE
Add descripition of a production deployment.

### PR DESCRIPTION
**Pull request by ewelinagr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -251,6 +253,178 @@ To run other tests:
 ./scripts/run_tests.sh
 ```
 
+## Deployment
+
+Instructions on how to set up the pipeline on a production environment.
```
Comment (gijskant) [inline README.md:258]:
```suggestion
Instructions on how to set up the pipeline on a production environment manually. The pipeline can also be installed with Puppet using the [pmc_pipeline](https://github.com/thehyve/pmc_pipeline) module.
```

File: README.md [hunk 2]
```diff
@@ -251,6 +253,178 @@ To run other tests:
...
+Create users `pmc` and `drop` with home directories:
+
+```shell
+sudo useradd -m -s /bin/bash pmc
+sudo useradd -m -s /bin/bash drop
+```
+
+If there is a list of users who should be able to log in as `drop` and/or `pmc` user through SSH,
+add /.ssh directories inside the newly created home directories of these users and put the list of SSH-RSA keys
+inside `authorized_keys` files (`home/pmc/.ssh/authorized_keys` and `home/drop/.ssh/authorized_keys`).
```
Comment (gijskant) [inline README.md:280]:
```suggestion
inside `authorized_keys` files (`/home/pmc/.ssh/authorized_keys` and `/home/drop/.ssh/authorized_keys`).
```

File: README.md [hunk 3]
```diff
@@ -251,6 +253,178 @@ To run other tests:
...
+
+
+#### Create a Python3 virtualenv
+
+```shell
+sudo -iu pmc
+cd /home/pmc
+python3 -m venv venv
+source venv/bin/activate
+python3 -m pip install -r /home/pmc/pmc-conversion/requirements/requirements.txt
```
Comment (gijskant) [inline README.md:354]: In the virtual environment the `pip` command is already set correctly, so you can use:
```suggestion
pip install -r /home/pmc/pmc-conversion/requirements/requirements.txt
```

File: README.md [hunk 4]
```diff
@@ -251,6 +253,178 @@ To run other tests:
...
+```
+
+#### Configure a Luigi deamon service
```
Comment (gijskant) [inline README.md:357]:
```suggestion
#### Configure a Luigi daemon service
```

File: README.md [hunk 5]
```diff
@@ -251,6 +253,178 @@ To run other tests:
...
+Then start the service:
+```shell
+sudo systemctl start luigi
+```
+
+and automatically get it to start on boot:
+```shell
+sudo systemctl enable luigi
+```
+
```
Comment (gijskant) [inline README.md:389]: Maybe add a 'Test the pipeline' step to manually trigger a data load?

File: README.md [hunk 6]
```diff
@@ -251,6 +253,178 @@ To run other tests:
...
+#### Prepare external tools
+
+Download the latest version of the transmart-copy.jar from the Nexus repository of The Hyve:
+```shell
+curl -f -L https://repo.thehyve.nl/service/local/repositories/releases/content/org/transmartproject/transmart-copy/17.2.8/transmart-copy-17.2.8.jar -o transmart-copy.jar
+```
+
+Then put the transmart-copy.jar inside `/home/pmc/libs`.
```
Comment (gijskant) [inline README.md:339]:
```suggestion
Download the latest version of the transmart-copy.jar from the Nexus repository of The Hyve to `/home/pmc/libs`:
```shell
curl -f -L https://repo.thehyve.nl/service/local/repositories/releases/content/org/transmartproject/transmart-copy/17.2.8/transmart-copy-17.2.8.jar -o /home/pmc/libs/transmart-copy.jar
```
```

File: README.md [hunk 7]
```diff
@@ -251,6 +253,178 @@ To run other tests:
...
+
+#### Prepare the configuration
+
+Prepare the [luigi.cfg](#luigi-configuration) and [email_config.cfg](#email-configuration)
```
Comment (gijskant) [inline README.md:343]: Could you also update these templates to have more reasonable defaults? E.g., `/home/drop/drop_zone` for `drop_dir`, `biomart_user` for `PGUSER`, `http://localhost:8081` for `transmart_url` and mark the values that (may) need to be changed with `# CHANGE ME`?

Comment (gijskant) [reply README.md:343]: It is clear that the pipeline was created a while ago ;)

File: README.md (hunk 1 above)
Comment (ewelinagr) [reply README.md:258]: This is still a private repository. Maybe we can discuss making it a public one.

Comment (gijskant) [reply README.md:258]: Good point. Good to leave it like it was.

**Comment by ewelinagr** [comment]
Still work in progress, to do:

- [ ] add a description on how to configure an offline user in Keycloak
- [ ] remove cBioPortal related code and config

**Review by gijskant — APPROVED** [review]

**Pull request merged**